packages/flutter/lib/src/cupertino/date_picker.dart

```diff
@@ -342,25 +342,40 @@ class _CupertinoDatePickerDateTimeState extends State<CupertinoDatePicker> {
   int textDirectionFactor;
   CupertinoLocalizations localizations;
 
-  // Alignment based on text direction. The variable name is self descriptive,
-  // however, when text direction is rtl, alignment is reversed.
+  /// Alignment based on text direction. The variable name is self descriptive,
+  /// however, when text direction is rtl, alignment is reversed.
   Alignment alignCenterLeft;
   Alignment alignCenterRight;
 
-  // Read this out when the state is initially created. Changes in initialDateTime
-  // in the widget after first build is ignored.
+  /// Read this out when the state is initially created. Changes in initialDateTime
```

Comment:
Same here. When the doc is in a 'final polished' form, we can have it in /// form. Otherwise, a // would remind us to revise this when we turn this private State subclass to a public one.